doc/src/guides/solving/solve-system-of-equations-algebraically.md

```diff
@@ -0,0 +1,153 @@
+>>> solutions
+[{x: -sqrt(6)*sqrt(z), y: -4*z}, {x: sqrt(6)*sqrt(z), y: -4*z}]
+```
+
+### Use a Solution Given as a Dictionary
+
+You can then extract solutions by indexing (specifying in brackets) the solution
+number, and then the symbol. For example `solutions[0][x]` gives the result for
+`x` in the first solution:
+
+```py
+>>> solutions[0][x]
+-sqrt(6)*sqrt(z)
+>>> solutions[0][y]
+-4*z
+```
+
+## Solve Results in a Set
+
+To get a list of symbols and set of solutions, use `set=True` instead of
+`dict=True`:
+
+```py
+from sympy import solve
+from sympy.abc import x, y, z
+solve([x**2 + y - 2*z, y + 4*z], [x, y], set=True)
+([x, y], {(-sqrt(6)*sqrt(z), -4*z), (sqrt(6)*sqrt(z), -4*z)})
+```
+
+## Options That Can Speed up {func}`~.solve`
+
+Refer to [](solving-guidance.md#options-that-can-speed-up).
+
+## Not All Systems of Equations Can be Solved
+
+### Systems of Equations With no Solution
+
+Some systems of equations have no solution. For example, the following two
+systems have no solution because they reduce to `1 == 0`:
+
+```py
+>>> from sympy import solve
+>>> from sympy.abc import x, y
+>>> solve([x + y - 1, x + y], [x, y], dict=True)
+[]
+```
+
+```py
+from sympy import solve
+from sympy.abc import x, y, z
+solve([x + y - (z + 1), x + y - z)], [x, y], dict=True)
+[]
+```
+
+The following system reduces to $z = 2z$, so it has no general solution, but it
+could be satisfied if $z=0$. Note that {func}`~.solve` will not assume that
+$z=0$ even though that is the only value of $z$ that makes the system of
+equations consistent because $z$ is a parameter rather than an unknown. That is,
+{func}`~.solve` does not treat $z$ as an unknown because it is not in the list
+of symbols specified as unknowns (`[x, y]`) and all such symbols are treated
+like parameters with arbitrary value. This is a semantic definition and has
+nothing to do with a symbol's creation (or import from {mod}`~.abc`).
+
+```py
+>>> from sympy import solve
+>>> from sympy.abc import x, y, z
+>>> solve([x + y - z, x + y - 2*z], [x, y], dict=True)
+[]
+```
+
+The following system is
+[overconstrained](https://en.wikipedia.org/wiki/Overdetermined_system), meaning
+there are more equations (three) than unknowns to be solved for (two, namely $x$
+and $y$), and has no solution:
+
+```py
+>>> from sympy import solve
+>>> from sympy.abc import x, y, z
+>>> solve([x + y - z, x - (z + 1), 2*x - y], [x, y], dict=True)
+[]
+```
+
+### Systems of Equations With no Closed-Form Solution
+
+Some systems of equations cannot be solved algebraically, for example those
+containing [transcendental
+equations](https://en.wikipedia.org/wiki/Transcendental_equation):
+
+```py
+>>> from sympy import cos, solve
+>>> from sympy.abc import x, y, z
+>>> solve([x - y, cos(x) - y], [x, y], dict=True)
+Traceback (most recent call last):
+    ...
+NotImplementedError: could not solve -y + cos(y)
+```
+
+So you can use {func}`~.nsolve` to find a numerical solution.
```

Comment:
Should we show this?

Reply:
Maybe if the same example is shown in the numerical page we can link to it there.